cli/src/app.rs

```diff
@@ -47,6 +47,15 @@ pub fn app() -> Command {
                 .action(ArgAction::SetTrue)
                 .long("no-check-certificate")
                 .help("Don't validate the server certificate when performing api requests"),
+            Arg::new("verbose")
+                .short('v')
+                .help("Increase the level of verbosity (the maximum is -vvv)")
+                .action(ArgAction::Count),
+            Arg::new("quiet")
+                .short('q')
+                .help("Reduce the level of verbosity (the maximum is -qq)")
+                .action(ArgAction::Count)
+                .conflicts_with("verbose"),
```

Comment:
If there is any `Arg` that deserves to be marked as [`global`](https://docs.rs/clap/latest/clap/builder/struct.Arg.html#method.global), this is probably it.

Otherwise, I expect users will end up in this loop:
```
# Run a command
phylum analyze foo.txt

# Some weird error happened?!? Add the verbose flag and try again.
phylum analyze -vv foo.txt

# Whoops... Can't put it there
phylum -vv analyze foo.txt
```